plugins/plugin-tests/tests/commands.rs

```diff
@@ -0,0 +1,140 @@
+use forc_index::commands::{
+    build::Command as BuildCommand, init::Command as InitCommand,
+    new::Command as NewCommand,
+};
+use std::path::{Path, PathBuf};
+use tempfile::{Builder, TempDir};
+
+#[test]
+fn init_command_creates_correct_project_tree() {
```

Comment:
It's convention that tests start with `test_` (checkout the tests in `fuel-indexer-tests`)